### ISSUE TITLE
[Filter Presets Quick Launcher] Feature Request

### ISSUE THREAD

**Issue by FregrinZ** [darklinkpower/PlayniteExtensionsCollection#426]
### Extension name

Filter Presets Quick Launcher

### Feature Description

I now made this mainly to filter by game launchers and works great , i have icons for steam,epic etc on top right screen the screen from your add on. I was wondering anyway to make right click option or something so can launch actual game launcher apps for example directly from playnite.

I apologise if this is wrong place, this all new to me and have no clue how find right places yet. Am looking on Reddit and googling as i go.

**Comment by darklinkpower** [comment]
Not possible, the reason is that extensions can give items to Playnite to add to the sidebar and topbar but basically can only define how they look and what they do when activated. It's not possible to add additional features like right clicking.

Although not for the top panel, you can already add launchers manually to the sidebar in Main menu -> Library -> Software tools

![image](https://github.com/darklinkpower/PlayniteExtensionsCollection/assets/1389286/3fe19484-820f-4138-a3f0-708b5af677b8)

**Comment by FregrinZ** [comment]
Hi,Thanks for the reply, shame its not possible. I didn’t add to side bar because have your filter on top row with icons and then on left side for launchers..Sent from my iPadOn 29 Aug 2023, at 13:38, Brandon ***@***.***> wrote:﻿
Not possible, the reason is that extensions can give items to Playnite to add to the sidebar and topbar but basically can only define how they look and what they do when activated. It's not possible to add additional features like right clicking.
Although not for the top panel, you can already add launchers manually to the sidebar in Main menu -> Library -> Software tools

**Comment by darklinkpower** [comment]
That's true but I don't know if I want to do all the work of implementing the control and also everything around it, like implementing the necessary UI, code, settings of the action, etc. This was an extension that I initially started creating for personal use and at least at the moment I'm not eager to put more work into it but never say never.
